build-system/server/new-server/transforms/css/css-transform.ts

```diff
@@ -45,8 +45,28 @@ interface StyleNode extends PostHTML.Node {
   content: string[]
 }
 
+function isStyleNode(node: PostHTML.Node | string): node is StyleNode {
+  if (node === undefined) {
+    return false;
+  }
+
+  if (typeof node === 'string') {
+    return false;
+  }
+
+  const styleNode = node as StyleNode;
+  return styleNode.tag === 'style';
```

Comment:
Nit: this seems simple enough that a one-liner would be readable enough here
```javascript
return node !== undefined &&
    typeof node !== 'string' &&
    (node as StyleNode).tag === 'style';
```

Reply:
done.

Reply:
For code like this that is never delivered to end-users, the readability is much better without the one-liner.

I'd vote for something like (if we're doing a nit).

```typescript
if (node === undefined || typeof node === 'string') {
  return false;
}

return (node as PostHTML.Node).tag === 'style';
```